### DISCUSSION TITLE
[Help] Removing absolute file path from the edit interactive mode

### DISCUSSION THREAD

**Discussion by ik1dial** [zk-org/zk#348] [Help] [answered]
### Verify

- [X] I searched the existing discussions for help

### How can we help you?

I am trying to remove the absolute path that seems to be automatically appended to each line in `zk edit --interactive` mode, no matter how I configure `fzf-line` parameter in the config file. 

Here's the how my interactive mode looks right now: 
![Pasted image](https://github.com/mickael-menu/zk/assets/87704058/e80742e1-3b78-4fc7-8cd6-92a31e934306)

The goal is to remove the absolute paths highlighted in red brackets (i've just migrated from logseq, that's why filepaths have it, but that shouldn't have any impact on the problem on hand)

Here's how  my config looks like: 
`fzf-preview = "bat -p --color always {-1}"`
`fzf-line = "{{style 'blue' filename-stem}}{{#each tags}} #{{this}}{{/each}} {{word-count}} {{style 'yellow' body}}"`

**Comment by mickael-menu** [comment, marked as answer]
You can't, because the absolute path is used to render the right-side preview. What you could do is add a bunch of spaces at the end of your `fzf-line` to hide it away.

https://github.com/mickael-menu/zk/blob/616eabb86e1d7add2934d139de19b5a0002eb1b8/internal/adapter/fzf/note_filter.go#L145-L148

**Reply by ik1dial** [reply]
Ah, gotcha, thank you!
